src/components/Todos.tsx

```diff
@@ -0,0 +1,107 @@
+import { useMemo } from 'react';
+import { SortBy, Todo } from '../types';
+
+type Props = {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  sortBy: SortBy,
+  handleDeleteTodo: (value: number) => void,
+  isLoading: boolean,
+  selectedTodo: number[]
+};
+
+export const Todos: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  sortBy,
+  handleDeleteTodo,
+  isLoading,
+  selectedTodo,
+}) => {
+  const filteredTodos = useMemo(() => {
+    if (sortBy === SortBy.all) {
+      return todos;
+    }
+
+    const isCompleted = sortBy === SortBy.completed;
+
+    return todos.filter(({ completed }) => completed === isCompleted);
+  }, [sortBy, todos]);
+
+  return (
+    <section className="todoapp__main">
+      {filteredTodos.map(todo => (
```

Comment:
[According to the guidelines](https://mate-academy.github.io/style-guides/htmlcss.html), we should let this component semantic using a `unordered list (ul)` instead of using `div`.